### PR TITLE
(Publish on 12/15/2023) Accounts: Clarify EOL for product-based pricing

### PR DESCRIPTION
**Pull request by rhetoric101**

### CODE REVIEW

File: src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/switch-new-models.mdx [hunk 1]
```diff
@@ -6,7 +6,11 @@ redirects:
...
+<Callout variant="important">
+  This doc references our original product-based pricing model. For more about pricing changes, see [Overview of our pricing models](/docs/accounts/accounts-billing/new-relic-one-pricing-users/transition-guide-new-pricing-plan/).
+</Callout>
+
+Some of our older customers may not have had an opportunity to update their pricing model yet and are still on our **original pricing model**. Our newer pricing model, sometimes referred to as the **New Relic One pricing model**, is much simpler and offers many benefits compared to our original pricing model ([learn more](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing)). This doc explains the options available for switching to the newer pricing model. 
 
 ## How free accounts switch
```
Comment (akristen) [inline src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/switch-new-models.mdx:15]: might need an anchor tag

File: src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-pricing-models.mdx [hunk 2]
```diff
@@ -22,8 +22,8 @@ import accountsBillingUI from 'images/accounts_screenshot-crop_billing-UI.webp'
...
 
-* Our primary usage-based pricing ([see docs](/docs/new-relic-one-pricing-billing)): this is our newer pricing model, which bills based on usage: data ingested and number of billable users. Almost all New Relic organizations are on this model. All organizations created on or after July 30, 2020 are on this model, as are some older organizations that have [switched to this model](/docs/accounts/original-accounts-billing/original-product-based-pricing/switch-new-models/). 
-* Our original product-based pricing model ([see docs](/docs/accounts/original-accounts-billing/product-pricing/product-based-pricing)): our older model involves subscriptions to specific products. 
+* Our primary usage-based pricing ([see docs](/docs/new-relic-one-pricing-billing)): This is our newer pricing model, which bills based on usage: data ingested and number of billable users. Almost all New Relic organizations are on this model. All organizations created on or after July 30, 2020 are on this model, as are some older organizations that have [switched to this model](/docs/accounts/original-accounts-billing/original-product-based-pricing/switch-new-models/).
```
Comment (akristen) [inline src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-pricing-models.mdx:25]:
```suggestion
* Our primary usage-based pricing ([see docs](/docs/new-relic-one-pricing-billing)): This is our newer pricing model, which bills based on data ingested and number of billable users. Almost all New Relic organizations are on this model. All organizations created on or after July 30, 2020 are on this model, as are some older organizations that have [switched to this model](/docs/accounts/original-accounts-billing/original-product-based-pricing/switch-new-models/). 
```

**Review by akristen — CHANGES_REQUESTED** [review]
Made a nitpicky copy change you can take or leave. The sentence had two colons: one for the beginning of the sentence and a second nested within it. You can keep your original if 'based on usage' is an important phrase we want customers to recognize. I also saw that the original doc is missing anchor tags. Could be worth going through and making sure the Markdown is all accounted for on that specific doc

File: src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/switch-new-models.mdx (hunk 1 above)
Comment (rhetoric101) [reply src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/switch-new-models.mdx:15]: @akristen for the link in line 13, we want users to start reading at the top, so I don't think there's a need for an anchor. The same applies for the link in line 10.

File: src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-pricing-models.mdx (hunk 2 above)
Comment (rhetoric101) [reply src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-pricing-models.mdx:25]: @akristen your removal of "usage" might help reduce confusion in the future. Let's go with that.

**Comment by rhetoric101** [comment]
@akristen could you double-check this and approve it?

**Review by akristen — APPROVED** [review]
Looks great!

**Pull request merged**